assets/_dashboard.scss

```diff
@@ -16,6 +16,11 @@ table.Dashboard {
     th:first-child {
         width: 50%;
     }
+    .gg-external {
```

Comment:
can we add this to the _icons.scss as a modifier ?

Reply:
it pertains to the icon in the context of the dashboard. I think it in the right place